src/component/modal/memberCreateModal/index.tsx

```diff
@@ -73,9 +94,34 @@ export default function MemberCreateModal({ open, onClose }: MemberInfoModalProp
     setMember({ ...member, [name]: formatPhoneNumber(value) });
   };
 
+  const uploadImage = async ({ presignedUrl, file }: { presignedUrl: string, file: File }) => {
+    await axios.put(presignedUrl, file, {
+      headers: {
+        'Content-Type': 'image/jpeg, image/png, image/svg+xml, image/webp',
+      },
+    }); // 헤더에 authrization을 담으면 안된다
+  };
+
+  const handleImage = async (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+
+    if (file) {
+      const presigned = await getPresignedUrl({
+        fileName: file?.name as string,
+      });
+
+      setImageInfo({ file, presignedUrl: presigned });
+    }
+  };
+
   const handleSave = () => {
     if (member) {
-      createMember(toMemberCreate(member));
+      if (imageInfo?.presignedUrl) {
+        uploadImage({ presignedUrl: imageInfo.presignedUrl.presignedUrl, file: imageInfo.file });
+        createMember(toMemberCreate(member));
+      } else {
+        createMember(toMemberCreate(member));
+      }
```

Comment:
```suggestion
      if (imageInfo?.presignedUrl) {
        uploadImage({ presignedUrl: imageInfo.presignedUrl.presignedUrl, file: imageInfo.file });
        }
        createMember(toMemberCreate(member));
```
개인적으로 else를 안쓰는 것이 보기 편해서 이렇게 적용할 수 있을 것 같아요!

Reply:
요건 로직이 잘못 작성했어서 if-else문에 같은 코드가 들어갔었네요😅
지금은 로직을 수정해서 if와 else가 다른 로직을 가지게 되었습니다.